kuma/static/styles/components/payments/_page.scss

```diff
@@ -73,82 +88,119 @@
+                    width: 45px;
+                }
 
-    .faqs {
-        flex: 60%;
+                .small {
+                    display: block;
+                    font-weight: 700;
+                }
+            }
 
-        a {
-            border-bottom: 1px solid $payment-faq-base-color;
-            color: $payment-faq-base-color;
-            padding-bottom: 1px;
-            text-decoration: none;
-        }
+            h3 {
+                font-size: 1.8em;
+                font-weight: normal;
+                margin-top: 0;
+            }
 
-        .faq {
-            background-color: #fff;
-            border: 2px solid $text-color;
-            border-radius: $payment-faq-border-radius;
-            margin: 2em;
-            padding: 48px;
-            position: relative;
-
-            ul {
-                list-style-type: initial;
-                padding-left: 1em;
+            @media #{$mq-tablet-and-up} {
+                > li {
+                    margin: 32px;
+                }
             }
+        }
 
-            p:last-child {
-                margin-bottom: 0;
+
+        > ul {
+            > li {
+                border: none;
+                padding: 24px 48px 48px;
             }
 
-            .faq-number {
-                background: #333;
-                box-sizing: border-box;
-                color: #fff;
-                font-family: $heading-font-family;
-                font-size: 36px;
-                font-weight: 500;
-                height: 45px;
-                left: -22px;
-                line-height: 40px;
-                padding: 0;
-                position: absolute;
-                text-align: center;
-                top: 42px;
-                width: 45px;
+            @media #{$mq-tablet-and-up} {
+                > li {
+                    margin: 0 32px;
+                }
             }
+        }
 
-            .small {
-                display: block;
-                font-weight: 700;
+        > form {
+            flex: 60%;
+            box-sizing: border-box;
+            padding: 24px 0 48px;
+            display: flex;
+            flex-direction: column;
+            @media #{$mq-tablet-and-up} {
+                padding-left: 80px;
+                padding-right: 80px;
+            }
+            input {
+                height: 40px;
+                border-width: 1px;
+                border-radius: $payment-faq-border-radius;
+            }
+            button {
+                color: $link-color;
+                border: 2px solid $link-color;
+                border-radius: $payment-faq-border-radius;
+                height: 40px;
+            }
+            .form-footer {
+                margin-top: 20px;
+                display: flex;
+                justify-content: space-between;
+                button {
+                    // this keeps button flush right
+                    margin-left: auto;
+                }
```

Comment:
Nit: There should be one empty line between each of these blocks.

Reply:
Fixed, thank you!